packages/compiler/src/app/builders/ng-app.builder.ts

```diff
@@ -342,11 +342,12 @@ export class NgAppBuilder {
       }
     }
 
-    // | styles.css
+    // | styles.scss
     if (diff.globalStyleChanged || _.has(diff, 'prev.globalStyle')) {
       writeFileSync(
-        path.join(srcDir, 'styles.css'),
-        prettier.format(this.globalStyle, { parser: 'css' }));
+        path.join(srcDir, 'styles.scss'),
+        this.globalStyle
```

Comment:
you can still run the code through prettier with the parser set to `css` as it was before. It will deal with SCSS just fine (see https://prettier.io/docs/en/options.html)

Reply:
Resolved!